client/index.html.tpl

```diff
@@ -76,12 +76,18 @@
 					<script async src="js/loading-error-handlers.js"></script>
 				</div>
 				<div id="chat-container" class="window">
+					<div id="chat-container-overlay"></div>
```

Comment:
Move this overlay to body, next to the image viewer. Keeping it here means you can't start dragging over the channel list.

And give this a better id like `upload-overlay` or something.

Reply:
I think you can start dragging over channel lost, as long as you drag on to chat container after, right? The onDrop listener is only on the chat container, not the channel list as well

Reply:
Yeah  change the onDrop to be on the entire document. It's rather weird UX to drag a file into Lounge and have it opened in that tab instead of being uploaded.

Reply:
done